examples/TemplateGenerationIterative.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>We should prefer the <code>snake_case</code>  form with the snake case filters, e.g. <code>OutputOrigin</code>  -&gt; <code>output_origin</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/HASI/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>